File: VB/XtraReport1.vb
Imports DevExpress.XtraReports.UI

Namespace Reporting_Export_Report_Groups_to_Separate_Excel_Sheets
	Partial Public Class XtraReport1
		Inherits DevExpress.XtraReports.UI.XtraReport

		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub CategoryNameLabel_BeforePrint(ByVal sender As Object, ByVal e As System.Drawing.Printing.PrintEventArgs) Handles CategoryNameLabel.BeforePrint
'INSTANT VB NOTE: The variable categoryNameLabel was renamed since Visual Basic does not handle local variables named the same as class members well:
			Dim categoryNameLabel_Renamed = DirectCast(sender, XRLabel)
			Dim categoryName = categoryNameLabel_Renamed.Text.Split("/"c)(0)
			categoryNameLabel_Renamed.Text = categoryName
			categoryNameLabel_Renamed.Value = categoryName
		End Sub
	End Class
End Namespace

File: CS/XtraReport1.cs
using DevExpress.XtraReports.UI;

namespace Reporting_Export_Report_Groups_to_Separate_Excel_Sheets {
    public partial class XtraReport1 : DevExpress.XtraReports.UI.XtraReport {
        public XtraReport1() {
            InitializeComponent();
        }

        private void CategoryNameLabel_BeforePrint(object sender, System.Drawing.Printing.PrintEventArgs e) {
            var categoryNameLabel = (XRLabel)sender;
            var categoryName = categoryNameLabel.Text.Split('/')[0];
            categoryNameLabel.Text = categoryName;
            categoryNameLabel.Value = categoryName;
        }
    }
}
